.travis/autotools-linux

```diff
@@ -33,6 +33,7 @@ travis_script() {
   add_config_flag --with-nacl-libs="$CACHEDIR/lib/amd64"
   add_config_flag --with-nacl-headers="$CACHEDIR/include/amd64"
   add_config_flag --disable-ipv6
+  add_config_flag --enable-test-network
```

Comment:
Let's remove this, so we can use the autotools build to ensure we can still connect to the old network.